index.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable */
+class Book {
+    constructor(title, author) {
+      this.title = title;
+      this.author = author;
+    }
+  }
```

Comment:
- [ ] Please don't disable linter errors instead fix them manually, It will help you to write more clean and professional code.

Reply:
hello how are you could you help me with this problem please hello how are you could you help me with this problem please